trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/hive/HiveSchemaPropertyConverter.java

```diff
@@ -44,9 +38,4 @@ public class HiveSchemaPropertyConverter extends PropertyConverter {
   public TreeBidiMap<String, String> engineToGravitinoMapping() {
     return TRINO_KEY_TO_GRAVITINO_KEY;
   }
-
-  @Override
-  public Map<String, PropertyEntry<?>> gravitinoPropertyMeta() {
```

Comment:
Why do we remove this method from `HiveSchemaPropertyConverter`?

Reply:
Because we remove `XXPropertiesMetaData` from `bundled catalog`, and `gravitinoPropertyMeta()` couldn't be used.

Reply:
I mean why do you removing Hive-related properties? I noticed that the meta classes for Hive properties are still included in the bundled-catalog project.

Reply:
`gravitinoPropertyMeta` is a abstract method, it is removed by Iceberg catalog, which will affect other catalogs.